static/css/styles.css

```diff
@@ -0,0 +1,31 @@
+.sidebar_nav {
+    width: 210px;
+    position: fixed;
+    z-index: 1;
+    top: 0;
+    left: 0;
+    background-color: yellow;
+    overflow-x: hidden;
+    padding-top: 20px;
+
+    font-family: "Roboto", "Lucida Grande", Verdana, Arial, sans-serif;
+}
+
+.sidebar_nav a {
+    padding: 6px 8px 6px 16px;
+    text-decoration: none;
+    font-size: 25px;
+    color: blue;
+    display: block;
+}
+
+.sidebar_nav a:hover {
+    color: red;
+}
+
+body {
+    margin-left: 210px;
+    padding: 0 10px;
+
+    font-family: "Roboto", "Lucida Grande", Verdana, Arial, sans-serif;
+}
```

Comment:
Blank line is missing

Reply:
It is a nit, but remember to do it always, so that github won't show this red sign